sphinx_gallery/utils.py

```diff
@@ -175,3 +175,12 @@ def _has_pypandoc():
         return None, None
     else:
         return True, version
+
+
+def _has_graphviz():
+    try:
+        import graphviz  # noqa F401
+    except ImportError:
```

Comment:
```suggestion
    except ImportError as exc:
```

Reply:
Sorry, you said where the import was but then the error said could not create graph so I assumed it was when the graph was being made